database/seed.js

```diff
@@ -41,7 +42,7 @@ let randomPhraseGenerator = function () {
   let returnedRandomString = '';
   let randomLength = Math.floor((randomNumberGenerator(20))); // Generates a random number between 0 - 20
   let randomWord = Math.floor((randomNumberGenerator(341)));
-  for (let i = 0; i < randomLength; i++) {
+  for (let i = 0; i < randomLength; i += 1) {
```

Comment:
why did you have to change all of these from `i++` to `i+=1` ?

Reply:
eslint for vscode doesn't like i++